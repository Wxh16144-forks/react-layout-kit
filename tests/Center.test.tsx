import { render } from '@testing-library/react';

import { Center } from 'react-layout-kit';

describe('Center', () => {
  it('默认', async () => {
    const { container } = render(<Center data-testid={'container'}>123</Center>);

    const node = container.firstChild;

    expect(node).toHaveStyle('display:flex');
    expect(node).toHaveStyle('align-items:center');
    expect(node).toHaveStyle('justify-content:center');
    expect(container).toMatchSnapshot();
  });
});
