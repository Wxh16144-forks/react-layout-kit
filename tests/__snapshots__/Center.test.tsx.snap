// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Center 默认 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div>
  <div
    class="layoutkit-center emotion-0"
    data-testid="container"
  >
    123
  </div>
</div>
`;
